dev-utils/test-config.js

```diff
@@ -118,10 +119,38 @@ function getBrowserList() {
     }
   }))
 }
+function parseVersion(version) {
+  const parts = version.split('.')
+  return {
+    full: version,
+    major: parts[0],
+    minor: parts[1],
+    patch: parts[2]
+  }
+}
+
+function isVersionInRange(version, min) {
+  var isInRange = true
+  if (version) {
+    var parsedVersion = parseVersion(version)
+    if (min) {
+      var minParsed = parseVersion(min)
+      isInRange =
+        isInRange &&
```

Comment:
not sure why we need this `isRange &&` check

Reply:
It's only because we might decide to add more condition before this check to expand this function.